### PR TITLE
Fix some hardcoded parameters

### PR DESCRIPTION
**Pull request by rouazana**

### CODE REVIEW

File: src/main/scala/com/linagora/openpaas/gatling/Configuration.scala [hunk 1]
```diff
@@ -11,7 +11,8 @@ object Configuration {
   val OpenPaaSHostName = Properties.envOrElse("OPENPAAS_HOSTNAME", "localhost")
   val OpenPaaSPort = Properties.envOrElse("OPENPAAS_PORT", "8080").toInt
   val OpenPaaSProtocol = Properties.envOrElse("OPENPAAS_PROTOCOL", "http")
-  val OpenPaaSBaseUrl = s"$OpenPaaSProtocol://$OpenPaaSHostName:$OpenPaaSPort"
+//  val OpenPaaSBaseUrl = s"$OpenPaaSProtocol://$OpenPaaSHostName:$OpenPaaSPort"
+  val OpenPaaSBaseUrl = s"$OpenPaaSProtocol://$OpenPaaSHostName"
```
Comment (huy-ta) [inline src/main/scala/com/linagora/openpaas/gatling/Configuration.scala:15]:
```suggestion
//  val OpenPaaSBaseUrl = s"$OpenPaaSProtocol://$OpenPaaSHostName:$OpenPaaSPort"
  val OpenPaaSBaseUrl = s"$OpenPaaSProtocol://$OpenPaaSHostName"
```

I think we can ditch `OpenPaaSHostName`, `OpenPaaSPort`, `OpenPaaSProtocol` altogether. Something like this is enough, isn't it?

```
val OpenPaaSBaseUrl = Properties.envOrElse("OPENPAAS_BASE_URL", "http://localhost:8080")
```

File: src/main/scala/com/linagora/openpaas/gatling/Configuration.scala [hunk 2]
```diff
@@ -11,7 +11,8 @@ object Configuration {
   val OpenPaaSHostName = Properties.envOrElse("OPENPAAS_HOSTNAME", "localhost")
   val OpenPaaSPort = Properties.envOrElse("OPENPAAS_PORT", "8080").toInt
   val OpenPaaSProtocol = Properties.envOrElse("OPENPAAS_PROTOCOL", "http")
-  val OpenPaaSBaseUrl = s"$OpenPaaSProtocol://$OpenPaaSHostName:$OpenPaaSPort"
+  private val usePortInUrl = Properties.envOrElse("OPENPAAS_PORT_IN_URL", "false").toBoolean
```
Comment (rouazana) [inline src/main/scala/com/linagora/openpaas/gatling/Configuration.scala:14]: why not simply detecting if the port is present?

**Review by rouazana — COMMENTED** [review]
I don't think adding a new (undocumented) option is necessary

File: src/main/scala/com/linagora/openpaas/gatling/Configuration.scala (hunk 2 above)
Comment (remk) [reply src/main/scala/com/linagora/openpaas/gatling/Configuration.scala:14]: I added the documentation

**Pull request merged**